<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomItem</class>
 <widget class="QWidget" name="RoomItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>55</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lb_roomID">
       <property name="text">
        <string>房间ID:10000001</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_pwd">
       <property name="text">
        <string>有密码</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_number">
       <property name="text">
        <string>5/6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_addRoom">
       <property name="text">
        <string>加入</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
